### PR TITLE
second PR

### PR DESCRIPTION
**Pull request by toocoolsachin**
## Attach Screenshot

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

**Comment by ni3-kdm** [comment]
> ## Description
> ## Attach Screenshot
> > Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

@toocoolsachin Can you add details in PR description template?

image

File: docs/TEAM.md [hunk 1]
```diff
@@ -84,3 +82,5 @@
 |                                                                 | Kingsley                                      | 3+ yrs of experience. Based in Nigeria and here to help :man_technologist:                         |
 |                                                                 | [Pratham](https://github.com/prasoonpratham)  | 2+ experience in JavaScript, always open to help :smile:                                           |
 |                                                                 | [Sarath](http://http://www.sarathnagaraj.ca/) | 6+ experience in JavaScript, Currently working on a side project using ReactJS :man_technologist:  |
+
+|<img src="" width="80px" height="80px"> | [Sachin](https://github.com/toocoolsachin) | I'm from India, experience in JavaScript, Node, React, Html, Css.
```
Comment (ni3-kdm) [inline docs/TEAM.md]: @toocoolsachin add "|" at the last and preview your changes.

image

**Review by Akshay2996 — APPROVED** [review]
After reviewing everything is looking fine👍

**Comment by RafaelBatman55** [comment]
Please check again your PR. There are some conflicts that must be resolved.

**Comment by proful** [comment]
pending since long

**Pull request closed without merging**